kernel/kernel-api/src/main/java/io/delta/kernel/types/StructField.java

```diff
@@ -119,21 +116,17 @@ public boolean isDataColumn() {
     @Override
     public String toString() {
         return String.format("StructField(name=%s,type=%s,nullable=%s,metadata=%s)",
-            name, dataType, nullable, "empty(fix - this)");
+            name, dataType, nullable, metadata.toString());
```

Comment:
When I saw `toString` here, I asked myself: what does FieldMetadata toString do? Well, it just calls `.toJson`

So why not jsut call metadata.toJson here?